### PR TITLE
Bug 1118732 - Improve performance of typing using the on-screen keyboard

### PR DESCRIPTION
**Pull request by davehunt**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[davehunt Dave Hunt (davehunt)](https://github.com/davehunt) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=adfd32219b33)

**Comment by bebef1987** [comment]
add hoc run http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/553/console

File: tests/python/gaia-ui-tests/gaiatest/gaia_test.py [hunk 1]
```diff
@@ -885,6 +885,9 @@ def cleanup_gaia(self, full_reset=True):
         # disable sound completely
         self.data_layer.set_volume(0)
 
+        # disable search suggestions
+        self.data_layer.set_setting('search.suggestions.enabled', False)
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/gaia_test.py]: +1 I think this might be one of the issues we saw some intermitent fails in the past

**Comment by davehunt** [comment]
This landed in https://github.com/mozilla-b2g/gaia/commit/3446a7b7aca75769981c1da7158895e85e7b6583

**Pull request closed without merging**